cocoa/NSPasteboard.go

```diff
@@ -98,10 +99,51 @@ func (pb NSPasteboard) ClearContents() {
 // SetStringForType sets the given string as the representation for the specified type for the first item on the receiver.
 // https://developer.apple.com/documentation/appkit/nspasteboard/1528225-setstring?language=objc
 func (pb NSPasteboard) SetStringForType(s string, t NSPasteboardType) {
-	pb.Send("setString:forType:", core.NSString_FromString(s),  core.NSString_FromString(t))
+	pb.Send("setString:forType:", core.NSString_FromString(s),  core.NSString_FromString(string(t)))
```

Comment:
we discussed using `core.String` instead of `core.NSString_FromString` and also documenting that using the shorthand is idiomatic for this project.